extensions/wikia/WallNotifications/WallNotifications.class.php

```diff
@@ -390,8 +389,7 @@ protected function sendEmails( array $watchers, WallNotificationEntity $notifica
 			if ( !empty( $mode ) && $watcher->getId() != 0 && (
 				( $mode == WALL_EMAIL_EVERY ) ||
 				( $mode == WALL_EMAIL_SINCEVISITED && empty( $this->uniqueUsers[$entityKey][$watcher->getId()] ) )
-			)) {
-
+			) && $watcher->getBoolOption( 'unsubscribed' ) !== true ) {
```

Comment:
We're actually doing this check inside of the Email extension [here](https://github.com/Wikia/app/blob/dev/extensions/wikia/Email/EmailController.class.php#L513-L517), so no need to check this.